행렬의-덧셈.js

```diff
@@ -0,0 +1,6 @@
+function solution(arr1, arr2) {
+    let result = arr1.map((item1, index1) =>
+        item1.map((item2, index2) => item1[index2] + arr2[index1][index2])
```

Comment:
```suggestion
    let result = arr1.map((el1D, row) =>
        el1D.map((el2D, col) => el1D[col] + arr2[row][col])
```
콜백함수의 매개변수도 의미있게 지어라